pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -171,7 +171,12 @@ def archive_pdf_file(pdf_archive_path, binary_content, extract_as_dict):
 
         time_info = (datetime.utcnow() + timedelta(hours=2)).strftime('%Y%m%d%H%M%S')  # UTC+2
```

Comment:
@marionb @michmuel I have no idea; I think that this definition exists propbbably since v1

Reply:
In case, all pdfs are stored a date without DST makes sense. But I think UTC would be more obvious than UTC+2. But there shouldn't be a disruption if someone is using it operationally. I just noticed this line and asked if there is a reason for this definition.

Reply:
The time in the docker container can be set. So this line is not general for all users.

Reply:
In general I would remove this +2 and solve it through the environment setting in the container. But this might be done in a separate PR.

Reply:
If this logic should be changed, it should indeed be in a separate issue - but since this was never reported as a problem so far, I would not prioritize that.

Reply:
Fine for me.

Reply:
for me its also fine and my ENV suggestion is only a suggestion. Since all users of pyramid_oereb are in the same timezone ;)

Reply:
By the way, it is the same with the creation_date. Depends also on your settings.

Reply:
FYI:
You can set an environment variable in the docker-compose like this and it should work

environment:
      - TZ=Europe/Zurich